src/components/GetDataDialog.tsx

```diff
@@ -19,45 +26,103 @@ function GetDataDialog({
   disableDownloadResultsButton: boolean;
   handleDownloadResultButtonClick: (identifier: string) => void;
 }) {
+  const DOCKER_RUN_COMMAND =
+    'docker run -t -v $(pwd):/data neurobagel/dataget:latest /data/cohort-participant-machine-results.tsv /data/output';
   const theme = useTheme();
   const fullScreen = useMediaQuery(theme.breakpoints.down('md'));
 
+  const [anchorEl, setAnchorEl] = React.useState<HTMLButtonElement | null>(null);
+  const [showPopover, setShowPopover] = useState(false);
+
+  const handleCopyClick = (event: React.MouseEvent<HTMLButtonElement>) => {
+    navigator.clipboard.writeText(DOCKER_RUN_COMMAND);
+    setAnchorEl(event.currentTarget);
+    setShowPopover(true);
+
+    setTimeout(() => {
+      setShowPopover(false);
+    }, 2000);
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
+    setShowPopover(false);
+  };
+
   return (
     <Dialog fullScreen={fullScreen} open={open} onClose={onClose} data-cy="get-data-dialog">
-      <DialogTitle>Example usage</DialogTitle>
       <DialogContent>
         <DialogContentText>
-          Please follow the steps below
+          <Typography variant="h6" className="font-bold">
+            Query results for programmatic use/download
+          </Typography>
+          To download data of matching subject-sessions for a query to your local filesystem, or to
+          use your query results further in other automated data workflows, we recommend using the
+          following machine-optimized version of your selected query results. This file contains
+          URIs instead of human-readable labels, making it ideal for scripts or integration with
+          other tools for linked or structured data:
```

Comment:
similarly, I'm noticing again how long this section currently is in the preview, which probably isn't ideal...

🍒  for simplification:
```suggestion
          Below is a machine-optimized version of your selected query results, which we recommend using as 
          input to scripts for downloading the data of matching subjects to your local filesystem. This file also 
          contains URIs instead of descriptive labels, making it ideal for integration with 
          other tools for linked or structured data:
```